python_tests/test_python_frontend.py

```diff
@@ -34,6 +35,19 @@
 
 RUN_NVFUSER = RUN_CUDA and not TEST_WITH_ROCM
 
+# This DEBUG_SERDE environment flag is used to debug serialization failures.
+#
+# 1) It disables automatically saving FusionCache upon program exit.
+#
+# 2) It resets the FusionCache after each test, which is useful for isolating
+# failures. Note, some failures only occur when running multiple tests
+# together and accumulating fusions in the cache.
+#
+# 3) It keeps the temporary files that are created during serde_check.
+# Normally, these files are deleted after each test.
+env_var_debug_serde = os.getenv("DEBUG_SERDE", None)
+debug_serde: bool = env_var_debug_serde in ("true", "1")
```

Comment:
nitpick: If this is only a helper for debug, wondering if it makes sense to have a finer control here? using debug_serde to turn on/off each of the three features.

i.e. if debug_serde & 4 -> disables automatically saving FusionCache upon program exit.

Reply:
That make sense. I think there are three options:

1. Disable automatic saving FusionCache on program exit. --- Option Name: "disable_atexit_cache_saving"
2. Disable accumulating fusions in FusionCache. --- Option Name: "single_fusion_only"
3. Both Options 1 and 2. --- Option Name: ["all", "true", "1"]